src/api/app/jobs/report_to_scm_job.rb

```diff
@@ -1,20 +1,19 @@
-class ReportToSCMJob < ApplicationJob
-  queue_as :scm # TODO: Is this fine?
+class ReportToScmJob < CreateJob
+  # We don't properly capitalize SCM in the class name since CreateJob is doing `CLASS_NAME.to_s.camelize.safe_constantize`
+  queue_as :scm
 
-  def perform
-    Event::Base.where(eventtype: ['Event::BuildFail', 'Event::BuildSuccess'], mails_sent: false).order(created_at: :asc).limit(1000).each do |event|
-      subscribers = event.subscribers
-      event.update(mails_sent: true) if subscribers.empty?
+  def perform(event_id)
+    event = Event::Base.find(event_id)
+    return true unless event
 
-      begin
-        event.subscriptions(:scm).each do |subscription|
-          SCMStatusReporter.new(subscription.payload, subscription.token.scm_token, event).call
-        end
-      rescue StandardError => e
-        Airbrake.notify(e, event_id: event.id)
-      ensure
-        event.update(mails_sent: true)
-      end
+    EventSubscription.joins('INNER JOIN events ON event_subscriptions.eventtype = events.eventtype')
```

Comment:
we have to filter here per project and/or package name

Reply:
Maybe we should have `package` & `project` columns in `Event` & `EventSubscription`?

Reply:
We came across this "need" a couple of time already.

Reply:
This is now handled with the changes we pushed today.